paraview/readme.txt

```diff
@@ -0,0 +1,27 @@
+1) setup ParaviewWeb
+    $ npm install paraviewweb  --save
+    $ npm install kw-web-suite --save-dev
+
+2) install the following packages
+    font-awesome
+    mout
+    json-bigint
+
+3) generate a single js file
+    $ ./node_modules/.bin/webpack
+
+4) start cloudberry service
+    $ sbt "project neo" "run"
+
+5) start a php server for the demo under "cloudberry/paraview/dist" directory
+    $ php -S [IP address]:[port]
+
+The demo should be available at http://[IP address]:[port]/
```

Comment:
Yes, the paraview/dist/CloudberryDemo.js is generated and I just removed it. I also updated the readme.